src/core/hw/gfxip/gfx9/gfx9UniversalCmdBuffer.cpp

```diff
@@ -6950,6 +6950,30 @@ void UniversalCmdBuffer::CmdSetPredication(
 
     uint32* pDeCmdSpace = m_deCmdStream.ReserveCommands();
 
+    // If the predicate is 32-bits, allocate a 64-bit embedded predicate, zero it, emit a ME copy from the
```

Comment:
Github won't let me put a comment on that line, but remove the assert on line 6925.